src/widgets/gauge.rs

```diff
@@ -81,6 +84,11 @@ impl<'a> Gauge<'a> {
         self.gauge_style = style;
         self
     }
+
+    pub fn use_unicode(mut self, unicode: bool) -> Gauge<'a> {
+        self.use_block = unicode;
```

Comment:
Any reason to have the property name be different from the name of the method setting its value ?

Reply:
Nope, just changed it over